examples/ref-templates/scripts/dynamic-fee.js

```diff
@@ -15,17 +15,25 @@ async function run(runtimeEnv, deployer) {
 	};
 	const contractName = "dynamic-fee.py";
 	// setup a contract account and send 1 ALGO from master
-	await deployer.fundLsigByFile(
-		contractName,
-		{ funder: masterAccount, fundingMicroAlgo: 100000000 },
-		{ closeRemainderTo: masterAccount.addr },
-		scInitParam
-	);
+	await deployer
+		.fundLsigByFile(
+			contractName,
+			{ funder: masterAccount, fundingMicroAlgo: 100000000 },
+			{ closeRemainderTo: masterAccount.addr },
+			scInitParam
+		)
+		.catch((error) => {
+			throw error;
+		});
```

Comment:
I don't follow. The error throwing functionality is already there in algob (and more verbose). Here we're just catching the thrown error, and re-throwing it. No?